plugin-build/src/test/kotlin/io/sentry/android/gradle/instrumentation/VisitorTest.kt

```diff
@@ -72,7 +72,7 @@ class VisitorTest(
         // and pass it through CheckClassAdapter to verify that the bytecode is correct and can be accepted by JVM
         val bytes = classWriter.toByteArray()
         val verifyReader = ClassReader(bytes)
-        val checkAdapter = CheckClassAdapter(ClassWriter(0), true)
+        val checkAdapter = CheckClassAdapter(ClassWriter(ClassWriter.COMPUTE_FRAMES), true)
```

Comment:
Had to change that as it seems the `CheckClassAdapter` implementation had changed underneath and does consider uncomputed frames now.